Merge Request: New Play - BMI Calculator

Adil-Khan-N opened the pull request:
Fixes #1524 

I’m excited to submit my merge request for the new BMI Calculator play! 🎈 This feature enhances our project by providing users with a simple and effective tool to calculate their Body Mass Index, making health and fitness tracking more accessible.

I would greatly appreciate it if you could review and accept this merge request, tagging it with the Hacktoberfest label. Your support means a lot to me!

Thank you for your time and consideration! 🙏

![image](https://github.com/user-attachments/assets/f06b3e8a-d167-4bfe-8589-c3675888e32b)

![image](https://github.com/user-attachments/assets/7064baa6-5a5f-433f-aaa0-1df835a6e122)

![image](https://github.com/user-attachments/assets/29d4d1ab-090a-4d7f-b2e8-b516afc5e58c)

priyankarpal commented on src/plays/bodymassindex-calculator/BodymassindexCalculator.js:
```diff
@@ -0,0 +1,105 @@
+  return (
+    <div className={`play-details ${isDarkMode ? 'dark-mode' : ''}`}>
+      <PlayHeader play={props} />
+      <div className="play-details-body">
```
please change the class name

priyankarpal commented on src/plays/bodymassindex-calculator/Readme.md:
```diff
@@ -0,0 +1,27 @@
+
+- User: Adil-Khan-N
+- Gihub Link: https://github.com/Adil-Khan-N
+- Blog: N/A
+- Video: N/A
+
+## Implementation Details
+
+Update your implementation idea and details here
+
+## Consideration
+
+Update all considerations(if any)
+
+## Resources
+
+Update external resources(if any)
```
remove these parts if you're not using

priyankarpal commented on src/plays/bodymassindex-calculator/styles.css:
```diff
@@ -0,0 +1,124 @@
+  }
+  
+  /* Background styling for the full page */
+  body {
+    background-color: #f8f9fa; /* Light background for outer area */
+  }
+  
+  .dark-mode body {
+    background-color: #1e1e1e; /* Dark background for outer area */
+  }
```
Please avoid targeting 'body' directly to prevent changing the default settings.

priyankarpal commented on src/plays/bodymassindex-calculator/styles.css:
```diff
@@ -0,0 +1,124 @@
+    background-color: #2c2c6c; /* Dark box background */
+  }
+  
+  .dark-mode input,
+  .dark-mode select {
```
avoid targeting `input` & `select` directly

priyankarpal commented:
@Adil-Khan-N please use different class names for the project. Also, the bg image is not rendering 

![Screenshot_2024-10-12_09-00-23](https://github.com/user-attachments/assets/c4d8737e-1176-43a5-9713-217852ee5ed1)

Adil-Khan-N commented:
I have changed the class names, but still the image is not rendering.

I have added a cover.jpg file in my play directory and also mentioned the name in the edit page of game, still it is not rendering, Could you please guide on what to give in the image URL section of the edit page.

![image](https://github.com/user-attachments/assets/c8ab052e-ceaa-42c9-b48d-b92c5dfb6404)

priyankarpal commented:
> I have changed the class names, but still the image is not rendering.
> 
> I have added a cover.jpg file in my play directory and also mentioned the name in the edit page of game, still it is not rendering, Could you please guide on what to give in the image URL section of the edit page.
> 
> ![image](https://private-user-images.githubusercontent.com/170500024/375928451-c8ab052e-ceaa-42c9-b48d-b92c5dfb6404.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5MTAyNTUsIm5iZiI6MTcyODkwOTk1NSwicGF0aCI6Ii8xNzA1MDAwMjQvMzc1OTI4NDUxLWM4YWIwNTJlLWNlYWEtNDJjOS1iNDhkLWI5MmM1ZGZiNjQwNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxNFQxMjQ1NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYWExMmU2OTcwMDdhMjk0OTk5MTllOTJkZmQ1ZWI5MjYxYWY3NzY2M2RlN2FjNTJkYWUzYTM3NmY2NzYwYTFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.imFjUUGaraRaiJV9jg3KalDnPfR5-aX7fRjgklS0Qyk)

for now merging this pr will check the cover image later.
